python/ray/autoscaler/_private/node_launcher.py

```diff
@@ -132,14 +125,28 @@ def _launch_node(
             self.node_provider_availability_tracker.update_node_availability(
                 node_type, launch_start_time, node_launch_exception
             )
-            # Do some special handling if we have a structured error.
-            self.log(
+            error_msg = (
                 f"Failed to launch {node_type}: "
                 f"({node_launch_exception.category}):"
                 f"{node_launch_exception.description}"
             )
+            self.event_summarizer.add(
```

Comment:
Maybe add a space after the colon on line 130
`f"({node_launch_exception.category}): "`